### ISSUE TITLE
Preference change for showing system addons (Irene)

### ISSUE THREAD

**Issue by irenesmith** [mdn/sprints#1392]
Copied from [Bugzilla #1544372](https://bugzilla.mozilla.org/show_bug.cgi?id=1544372)

Hi Irene! We are slightly modifying a behavior currently documented on MDN at https://developer.mozilla.org/en-US/docs/Tools/about:debugging#Opening_the_aboutdebugging_page

    Whether or not system add-ons appear in the list on this page depends on the setting of the devtools.aboutdebugging.showSystemAddons preference. If you need to see system add-ons, navigate to about:config and make sure that this value is set to true.

In Firefox 68, the preference will be renamed devtools.aboutdebugging.showHiddenAddons. The behavior for the old about:debugging remains the same. For the new about:debugging however this preference will now toggle the visibility of both "System addons" and "hidden" webextensions. We can't really expect regular users to know what both of those concepts are so maybe we should call them "internal extensions (system add-ons and hidden extensions)". Maybe the extensions team has coined a better term for them?

# Acceptance criteria
- [x] about:debugging page describes new behavior
- [x] New text has been reviewed
- [x] Information about changed preference added to 68 release notes

**Comment by irenesmith** [comment]
Added the following comment to the release notes:

- The preference that controls the visibility of internal extensions (system add-ons and hidden extensions) on the about:debugging page has been changed from devtools.aboutdebugging.showSystemAddons to devtools.aboutdebugging.showHiddenAddons (bug 1544372).

I also changed [that section of about:debugging](https://developer.mozilla.org/en-US/docs/Tools/about:debugging#This_Firefox) to reference the new value. The old version of the page still has the original value.
